src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const rentPrice = 40;
+  const discount1 = 20;
+  const discount2 = 50;
```

Comment:
```suggestion
  const smallDiscount = 20;
  const bigDiscount = 50;
```
1 и 2 мне ни о чем не говорят, и я не узнаю больше о содержимом переменной